### PR TITLE
feat(website): add Tweets section

### PR DESCRIPTION
**Pull request by yangshun**
## Motivation

As part of revamping the homepage (#6386), we're add some testimonials.

Maybe we shouldn't be showing tweets from the core team? But I'm a user myself and I like it :S

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

image

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Our Lighthouse is too harsh😝 The Twitter handles and dates don't have enough contrast ratios

**Comment by yangshun** [comment]
> The Twitter handles and dates don't have enough contrast ratios

Wew changing the color a little worked. Thanks for the tip!

**Comment by Josh-Cena** [comment]
Love the design! I wonder if we should make the Tweet's header direct to the author's profile? Maybe make the links in the text clickable as well? 🤔

**Comment by yangshun** [comment]
> Love the design! I wonder if we should make the Tweet's header direct to the author's profile? Maybe make the links in the text clickable as well? 🤔

I'm directly ripping off turborepo.org and I think their intention was to keep people on the page so the bounce rate isn't that high, which I don't disagree with haha.

Gosh GitHub needs replies on comments.

**Comment by slorber** [comment]
That looks nice yeah

Personally, I like to be able to click on testimonials and see they are legit, as anyone can easily write anything anywhere these days. I find that the link on the date is not so obvious, but that can be a good compromise...

I feel like we should focus on including testimonials that say something positive about Docusaurus. Launching a site is not a good-enough positive feedback to me, unless it's from someone widely known. I also find it distracting to see a long text with things like "thank you x y z", "we migrated to x" etc.... I'll try to create a new collection of my favorite picks

**Comment by slorber** [comment]
Here's my collection filtered: https://twitter.com/sebastienlorber/timelines/1484541096797605888
(still big 😅 )


I like the idea of having good diversity on testimonials. 

If we can feature popular projects, frontend devs, web3 devs, indie hackers, academics, technical writers, large companies... 
And also showcase a bit the value proposition of docusaurus

I think it's better than "A deployed B with Docusaurus, congratz C"

Popular devs

- banteg: https://twitter.com/bantg/status/1463608561368457225
- Mark Erikson: https://twitter.com/acemarke/status/1452725153998245891
- Mael Nison: https://twitter.com/arcanis/status/1351620354561732608
- Max Lynch: https://twitter.com/maxlynch/status/1375113166007455748
- Debbie O'Brien: https://twitter.com/debs_obrien/status/1374615572298801155
- Paul Armstrong: https://twitter.com/paularmstrong/status/1387059593373700100
- Sanket Sahu: https://twitter.com/sanketsahu/status/1328677366642528257
- Nader Dabit: https://twitter.com/dabit3/status/1394685348375052295
- Swyx: https://twitter.com/swyx/status/1418405515684581378
- Rachel Nabors: https://twitter.com/rachelnabors/status/1478490902037467137

Diverse projects:
- Supabase: https://twitter.com/supabase/status/1328960757149671425
- Electron: https://twitter.com/electronjs/status/1450862945139789824
- Redux: https://twitter.com/pavanpodila/status/1313026881730473984
- Mobx.dart: https://twitter.com/pavanpodila/status/1313026881730473984
- IOTA: https://twitter.com/Dr_Electron/status/1443635328376508433

Features:
- i18n: https://twitter.com/clairefroe/status/1369479424274440192
- i18n + flexibility: https://twitter.com/tinkertim/status/1423358665726304260
- versioning: https://twitter.com/fouadmatin/status/1430274645685522436
- get started fast: https://twitter.com/erchwy/status/1452414432320139268
- get started fast: https://twitter.com/dested/status/1392128025198616580
- flexibility: https://twitter.com/theshalvah/status/1400778320460107777

Simple and effective:
- https://twitter.com/BrodasGeo/status/1435649345857019904
- https://twitter.com/alirz_sedghi/status/1440794110718197768
- https://twitter.com/MrMace_/status/1458724901225844738
- https://twitter.com/zachcodes/status/1478057131345653764

**Comment by yangshun** [comment]
That's a huge list. Which shall we show on the homepage? I can make the change to the homepage and create a separate page for the rest of the tweets.

**Comment by slorber** [comment]
That looks like a nice list for me, still probably a bit too long 🤷‍♂️ 

Popular devs:
- Mark Erikson: https://twitter.com/acemarke/status/1452725153998245891
- Mael Nison: https://twitter.com/arcanis/status/1351620354561732608
- Max Lynch: https://twitter.com/maxlynch/status/1375113166007455748
- Kent C. Dodds: https://twitter.com/kentcdodds/status/1323806816019468288
- Debbie O'Brien: https://twitter.com/debs_obrien/status/1374615572298801155
- Paul Armstrong: https://twitter.com/paularmstrong/status/1387059593373700100
- Sanket Sahu: https://twitter.com/sanketsahu/status/1328677366642528257
- Swyx: https://twitter.com/swyx/status/1418405515684581378
- Rachel Nabors: https://twitter.com/rachelnabors/status/1478490902037467137
- Nader Dabit: https://twitter.com/dabit3/status/1394685348375052295
- John Reilly: https://twitter.com/johnny_reilly/status/1469238609266028545
- banteg: https://twitter.com/bantg/status/1463608561368457225

Diverse projects:
- Supabase: https://twitter.com/supabase/status/1328960757149671425
- Electron: https://twitter.com/electronjs/status/1450862945139789824
- IOTA: https://twitter.com/Dr_Electron/status/1443635328376508433

- i18n + flexibility: https://twitter.com/tinkertim/status/1423358665726304260

Are we allowed to shorten a bit the quotes?

I've used this project in the past to render static tweets efficiently based on some AST: https://github.com/transitive-bullshit/react-static-tweets

We could eventually automate our wall of love

File: website/src/data/tweets/index.tsx [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    avatar:
+      'https://pbs.twimg.com/profile_images/823614982394769408/C4KgET17_400x400.jpg',
+    content: (
+      <>
+        Continue to be impressed and excited about Docusaurus v2 alpha releases.
+        Already using the sidebar items generator to help monorepo workspace
+        devs create their own doc pages without any configuration needed.
+      </>
+    ),
+    showOnHomepage: true,
```
Comment (Josh-Cena) [inline website/src/data/tweets/index.tsx:96]: I assume we will have a separate "testimonials" page in the future?

Comment (yangshun) [reply website/src/data/tweets/index.tsx:96]: Yes one page with more categories of Tweets according to [this](https://github.com/facebook/docusaurus/pull/6426#issuecomment-1018644968)

**Review by slorber — APPROVED** [review]
Looks good to me 👍 ready to merge?

**Pull request merged**